src/contact.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta
+			name="viewport"
+			content="width=device-width, initial-scale=1.0"
+		/>
+		<title>Contact</title>
+	</head>
+	<body>
+		<h1>Contact</h1>
+		<form id="contactForm">
+			<label for="name">Name:</label>
+			<input
+				type="text"
+				id="name"
+				name="name"
+				required
+			/>
+
+			<label for="email">Email:</label>
+			<input
+				type="email"
+				id="email"
+				name="email"
+				required
+			/>
+
+			<label for="message">Message:</label>
+			<textarea
+				name="message"
+				id="message"
+				required
+			></textarea>
+
+			<button type="submit">Submit</button>
+		</form>
+
+		<script>
+			document.getElementById('contactForm').addEventListener('submit', (e) => {
+				e.preventDefault();
+

```

Comment:
I would also reccomend to add some input validation to ensure that correct data is sent in correct format.